webpack.config.js

```diff
@@ -0,0 +1,21 @@
+var path = require('path')
+var webpack = require('webpack')
+var UnminifiedWebpackPlugin = require('unminified-webpack-plugin')
+
+module.exports = {
+  entry: {
+    index: './browser.js'
+  },
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'promptypay-browser.min.js'
```

Comment:
We can use `output.library` to generate a UMD library (works with CommonJs, AMD, and globals.)

For more details: https://webpack.js.org/configuration/output/#output-librarytarget